locales/en.json

```diff
@@ -464,24 +464,18 @@
   "Public or non-profit long-term care home": "Public or non-profit long-term care home",
   "Did you pay property tax?": "Did you pay property tax?",
   "Property tax paid": "Property tax paid",
+  "Invalid value": "Invalid value",
+  "$999.00": "$999.00",
+  "$0.00": "$0.00",
```

Comment:
Hmm, do you know where these three values are coming from? They all look like they shouldn't be here to me.